tensorflow/core/kernels/mkl_maxpooling_op.cc

```diff
@@ -276,10 +276,12 @@ class MklMaxPoolingGradOp : public OpKernel {
     mkl_context.pooling_res[dnnResourceDiffSrc] = const_cast<void*>(
         static_cast<const void*>(output_tensor->flat<T>().data()));
 
+#if 0
```

Comment:
Please just delete these sections of code between #if and #endif (same for below).